sentence_transformers/SentenceTransformer.py

```diff
@@ -576,6 +584,8 @@ def encode(
             device (str, optional): Which :class:`torch.device` to use for the computation. Defaults to None.
             normalize_embeddings (bool, optional): Whether to normalize returned vectors to have length 1. In that case,
                 the faster dot-product (util.dot_score) instead of cosine similarity can be used. Defaults to False.
+            truncate_dim (int, optional): The dimension to truncate sentence embeddings to. `None` means we will used the value of the
+                model's config. Defaults to None.
```

Comment:
```suggestion
        truncate_dim (int, optional): The dimension to truncate sentence embeddings to. If `None`, no truncation is applied.
            Truncation is especially interesting for `Matryoshka models <https://sbert.net/examples/sentence_transformer/training/matryoshka/README.html>`_,
            i.e. models that are trained to still produce useful embeddings even if the embedding dimension is reduced.
            Truncated embeddings require less memory and are faster to perform retrieval with, but note that inference
            is just as fast, and the embedding performance is worse than the full embeddings. Defaults to None.
```
Perhaps we can use this for `SentenceTransformer` `__init__`, `encode`, and `encode_multi_process`?
I think it's important to explain why someone would ever do this, as that wasn't clear at all with my original docstrings.